### PR TITLE
Feature/articles

### PR DESCRIPTION
**Pull request by mgkyawzayya**

### CODE REVIEW

File: app/Http/Controllers/Api/Article/ArticleApiController.php [hunk 1]
```diff
@@ -2,23 +2,59 @@
 
 namespace App\Http\Controllers\Api\Article;
 
-use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
+use App\Modules\Articles\Models\Article;
```
Comment (setkyar) [inline app/Http/Controllers/Api/Article/ArticleApiController.php:8]: Please sort namespace by length.

File: app/Http/Controllers/Api/Article/ArticleApiController.php [hunk 2]
```diff
@@ -2,23 +2,59 @@
...
-                'hello' => 'World'
-            ]
-        ];
+        $articles = Article::latest()->paginate(20);
+
+        $articles = Fractal::create()
+            ->collection($articles->getCollection())
+            ->transformWith(new ArticleIndexTransformer())
+            ->paginateWith(new IlluminatePaginatorAdapter($articles))
+            ->toArray()
```
Comment (setkyar) [inline app/Http/Controllers/Api/Article/ArticleApiController.php]:
```suggestion
            ->toArray();
```

File: app/Http/Controllers/Api/Article/ArticleApiController.php [hunk 3]
```diff
@@ -2,23 +2,59 @@
...
+        ;
```
Comment (setkyar) [inline app/Http/Controllers/Api/Article/ArticleApiController.php]:
```suggestion
```

File: app/Http/Controllers/Api/Article/ArticleApiController.php (hunk 1 above)
Comment (mgkyawzayya) [reply app/Http/Controllers/Api/Article/ArticleApiController.php:8]: Ok bro

File: app/Http/Controllers/Api/Article/ArticleApiController.php [hunk 4]
```diff
@@ -2,23 +2,59 @@
...
+    /**
+     * Store a new article.
+     *
+     * @param Request $request
+     *
+     * @return \Illuminate\Contracts\Routing\ResponseFactory|\Illuminate\Http\Response
+     */
+    public function store(StoreArticle $request)
+    {
+        // Use form request to validate (https://laravel.com/docs/6.0/validation#form-request-validation)
```
Comment (setkyar) [inline app/Http/Controllers/Api/Article/ArticleApiController.php]: Remove comments, that does not need anymore.

File: app/Modules/Articles/Models/Article.php [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Modules\Articles\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Article extends Model
+{
+    protected $fillable = ['title', 'body'];
```
Comment (setkyar) [inline app/Modules/Articles/Models/Article.php:12]: Add doc block comment.

File: app/Modules/Articles/Transformers/ArticleIndexTransformer.php [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Modules\Articles\Transformers;
+
+use League\Fractal\TransformerAbstract;
+use App\Modules\Articles\Models\Article;
+
+class ArticleIndexTransformer extends TransformerAbstract
+{
+    public function transform(Article $article)
```
Comment (setkyar) [inline app/Modules/Articles/Transformers/ArticleIndexTransformer.php:17]: Add doc_block comment.

File: app/Modules/Articles/Transformers/ArticleShowTransformer.php [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Modules\Articles\Transformers;
+
+use League\Fractal\TransformerAbstract;
+use App\Modules\Articles\Models\Article;
+
+class ArticleShowTransformer extends TransformerAbstract
+{
+    public function transform(Article $article)
```
Comment (setkyar) [inline app/Modules/Articles/Transformers/ArticleShowTransformer.php]: Add doc_block comment.

File: database/seeds/DatabaseSeeder.php [hunk 8]
```diff
@@ -6,11 +6,10 @@ class DatabaseSeeder extends Seeder
 {
     /**
      * Seed the application's database.
-     *
-     * @return void
```
Comment (setkyar) [inline database/seeds/DatabaseSeeder.php]: Why you removed the comment?

**Review by setkyar — COMMENTED** [review]
@mgkyawzayya I added few comments to update, please take a check.

File: database/seeds/DatabaseSeeder.php (hunk 8 above)
Comment (mgkyawzayya) [reply database/seeds/DatabaseSeeder.php]: Sorry I will add now :('

File: app/Modules/Articles/Models/Article.php [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Modules\Articles\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Article extends Model
+{
+    /** Article Model
```
Comment (setkyar) [inline app/Modules/Articles/Models/Article.php]:
```suggestion
    /**
```

File: app/Modules/Articles/Transformers/ArticleIndexTransformer.php [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace App\Modules\Articles\Transformers;
+
+use League\Fractal\TransformerAbstract;
+use App\Modules\Articles\Models\Article;
+
+class ArticleIndexTransformer extends TransformerAbstract
+{
+    /**
+     * Show All Article
```
Comment (setkyar) [inline app/Modules/Articles/Transformers/ArticleIndexTransformer.php]:
```suggestion
     * Transform articles to an array
     *
```

**Review by setkyar — COMMENTED** [review]
@mgkyawzayya Don't need to rush, take your time and be careful the detail. I added new comments, take a check and update.

File: app/Modules/Articles/Transformers/ArticleIndexTransformer.php [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace App\Modules\Articles\Transformers;
+
+use League\Fractal\TransformerAbstract;
+use App\Modules\Articles\Models\Article;
+
+class ArticleIndexTransformer extends TransformerAbstract
+{
+    /**
+     * Transform articles to an array
```
Comment (setkyar) [inline app/Modules/Articles/Transformers/ArticleIndexTransformer.php:11]:
```suggestion
     * Transform articles to an array
     *
```

**Review by setkyar — APPROVED** [review]
Finally approved 🎉

**Pull request merged**